basisu_tool.cpp

```diff
@@ -85,6 +85,7 @@ static void print_usage()
 		" -no_alpha: Always output non-alpha basis files, even if one or more inputs has alpha\n"
 		" -force_alpha: Always output alpha basis files, even if no inputs has alpha\n"
 		" -separate_rg_to_color_alpha: Separate input R and G channels to RGB and A (for tangent space XY normal maps)\n"
+		" -swizzle rgba: Apply a general swizzle operation to the incoming colour channels\n"
```

Comment:
You probably ought to use American spelling here.

Reply:
Good point, done